homework/45169/python2html.flex

```diff
@@ -0,0 +1,66 @@
+%{
+#include <stdio.h>
+%}
+
+DIGIT    [0-9]
+ID       [a-zA-Z_$][a-zA-Z0-9_$]*
+STRQ     '('{2})?([^\']|(\\\'))*'('{2})?
+STRDQ    \"(\"{2})?([^\"]|(\\\"))*\"(\"{2})?
```

Comment:
Could have been simpler if you don't try to handle escaped quotes inside.
You don't need that since the lexer will look for the longest match possible

Reply:
I did better string matching in HW2